src/VisualStudio/Next/Remote/RemoteHostClientFactory.cs

```diff
@@ -8,6 +8,8 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
+    using Workspace = Microsoft.CodeAnalysis.Workspace;
```

Comment:
The AnyCode thing also has a type called 'Workspace'? Since our use of those type names is low, are we better off just doing a extern alias rather than making that viral?